frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeList/CodeLists/EditCodeList/EditCodeList.tsx

```diff
@@ -0,0 +1,69 @@
+import type { CodeList, CodeListEditorTexts } from '@studio/components';
+import { StudioCodeListEditor, StudioToggleableTextfield } from '@studio/components';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import type { CodeListWithMetadata } from '../../CodeList';
+import { useOptionListEditorTexts } from '../../hooks/useCodeListEditorTexts';
+import { KeyVerticalIcon } from '@studio/icons';
+import { FileNameUtils } from '@studio/pure-functions';
+import { useInputCodeListNameErrorMessage } from '../../hooks/useInputCodeListNameErrorMessage';
+
+export type EditCodeListProps = {
+  codeList: CodeListWithMetadata;
+  onUpdateCodeListId: (codeListId: string, newCodeListId: string) => void;
+  onUpdateCodeList: (updatedCodeList: CodeListWithMetadata) => void;
+  codeListNames: string[];
```

Comment:
The variable is only used for validation and is therefore misleading
```suggestion
  invalidCodeListNames: string[];
```

Reply:
Hm, I'm not sure if I agree on this one. In the context of the CodeListPage this variable is only the names, but for the  findFileNameError-function it is treated as a list of invalid names. But can change if you strongly disagree 🫡